tailwind.config.ts

```diff
@@ -30,6 +30,22 @@ module.exports = {
           900: '#234E99',
         },
       },
+      spacing: {
+        '11': '2.75rem',
+        '12': '3rem',
+        '13': '3.25rem',
+        '14': '3.5rem',
+        '15': '3.75rem',
+        '16': '4rem',
+        '17': '4.25rem',
+        '18': '4.5rem',
+        '19': '4.75rem',
+        '20': '5rem',
+        '46': '11.5rem',
+      },
+      fontSize: {
+        '26px': '26px',
```

Comment:
I think 🤔 '26px': '26px'   won't work.
You can use text-[26px] in every line instead or redifain xl from 24px to 26px if in Figma layout everywhere use 26px